tests/support_server/wait_children.pm

```diff
@@ -26,11 +26,16 @@ use base 'basetest';
 use testapi;
 use lockapi;
 use mmapi;
+use version_utils 'is_microos';
 
 sub run {
     my $self = shift;
 
-    select_console 'root-console';
+    if (is_microos) {
```

Comment:
This looks weird and needs at least a comment explaining why this is necessary

Reply:
I am not sure why, but when we set `select_console 'root-console';` in `DESKTOP=gnome` it's not switching to the console.
So, an example without this change:
http://fromm.arch.suse.de/tests/548#step/wait_children/1 
and with this change:
https://openqa.opensuse.org/tests/1666901#step/wait_children/1

Reply:
Probably because gnome is running in the wrong tty. You could boot with `3`/`systemd.unit=multi-user.target` added to `EXTRABOOTPARAMS` and force `DESKTOP` to `textmode` to work around that.

Though creating a more sensible support server without desktop would be great...

Reply:
> Probably because gnome is running in the wrong tty. You could boot with `3`/`systemd.unit=multi-user.target` added to `EXTRABOOTPARAMS` and force `DESKTOP` to `textmode` to work around that.
> 
I will try that

> Though creating a more sensible support server without desktop would be great...
Yes, I agree, having a "service" machine with useless desktop is actually useless.. but it's the way it's done today for TW ssh installation as well...

Reply:
Eventually, it works with `EXTRABOOTPARAMS_BOOT_LOCAL=3` and `DESKTOP=textmode`. Thanks a lot for your help!